django_ibmi/operations.py

```diff
@@ -510,22 +473,20 @@ def year_lookup_bounds_for_date_field(self, value):
     def bulk_insert_sql(self, fields, num_values):
         values_sql = "( %s )" % (", ".join(["%s"] * len(fields)))
         if isinstance(num_values, int):
-            bulk_values_sql = "VALUES " + \
-                ", ".join([values_sql] * (num_values))
+            bulk_values_sql = "VALUES " + ", ".join([values_sql] * num_values)
         else:
-            bulk_values_sql = "VALUES " + \
-                ", ".join([values_sql] * len(num_values))
+            bulk_values_sql = "VALUES " + ", ".join([values_sql] * len(num_values))
         return bulk_values_sql
 
-    def for_update_sql(self, nowait=False):
+    def for_update_sql(self, nowait=False, skip_locked=False, of=()):
         # DB2 doesn't support nowait select for update
         if nowait:
             raise utils.DatabaseError(
                 "Nowait Select for update not supported ")
         else:
             return 'WITH RS USE AND KEEP UPDATE LOCKS'
```

Comment:
We should probably throw exceptions if skip_locked, of, or no_key is specified. (no_key is in the latest Django: https://github.com/django/django/blob/76c0b32f826469320c59709d31e2f2126dd7c505/django/db/backends/base/operations.py#L203)